_summaries/kingma2014adam_2.md

```diff
@@ -1,19 +1,28 @@
 ---
 layout: summary
 title: Summary
-paper: {{paper_tag}}
+paper: kingma2014adam
 # Please fill out info below
-author: # Your GitHub id
-score: # How did you like this paper 0(dislike) to 10(love)
+author: ishank-arora
+score: 9
 ---
 
-TODO: Summarize the paper:
 * What is the core idea?
+  * Introduces the Adam (Adaptive Moment Estimation) algorithm.
```

Comment:
I suggest pointing out the motivation of the paper, i.e., the problem of existed methods and why are they important. Something like: SFO is not memory efficient, or RMS does not have a bias-correction term.